sdk/python/kfp/components/_component_store.py

```diff
@@ -91,3 +92,24 @@ def load_component(self, name, digest=None, tag=None):
                 return comp._load_component_from_yaml_or_zip_bytes(response.content, url, component_ref)
 
         raise RuntimeError('Component {} was not found. Tried the following locations:\n{}'.format(name, '\n'.join(tried_locations)))
+
+    def _load_component_from_ref(self, component_ref : ComponentReference) -> Callable:
```

Comment:
Nitpick, extra whitespace between variable name and `:`

Reply:
Fixed.